src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php

```diff
@@ -16,6 +16,9 @@
 
 class ContainerAwareEventManagerTest extends \PHPUnit_Framework_TestCase
 {
+    protected $container;
+    protected $evm;
```

Comment:
use private vibility please (and this should be submitted separately as a bugfix as it should not go only in master)